diff --git a/platform_testing/build/tasks/tests/instrumentation_test_list.mk b/platform_testing/build/tasks/tests/instrumentation_test_list.mk
index 977fb0b..46f6789 100644
--- a/platform_testing/build/tasks/tests/instrumentation_test_list.mk
+++ b/platform_testing/build/tasks/tests/instrumentation_test_list.mk
@@ -55,7 +55,6 @@ instrumentation_tests := \
     UsbTests \
     DownloadProviderTests \
     EmergencyInfoUnitTests \
-    CalendarProviderTests \
     SettingsLibTests \
     RSTest \
     PrintSpoolerOutOfProcessTests \

diff --git a/platform_testing/build/tasks/tests/instrumentation_test_list.mk b/platform_testing/build/tasks/tests/instrumentation_test_list.mk
index 23861c8..977fb0b 100644
--- a/platform_testing/build/tasks/tests/instrumentation_test_list.mk
+++ b/platform_testing/build/tasks/tests/instrumentation_test_list.mk
@@ -74,9 +74,3 @@ instrumentation_tests := \
     FrameworksNetSmokeTests\
 
 
-# Storage Manager may not exist on device
-ifneq ($(filter StorageManager, $(PRODUCT_PACKAGES)),)
-
-instrumentation_tests += StorageManagerUnitTests
-
-endif

diff --git a/platform_testing/build/tasks/tests/platform_test_list.mk b/platform_testing/build/tasks/tests/platform_test_list.mk
index 7ca9047..4592368 100644
--- a/platform_testing/build/tasks/tests/platform_test_list.mk
+++ b/platform_testing/build/tasks/tests/platform_test_list.mk
@@ -23,7 +23,6 @@ platform_tests += \
     BootHelperApp \
     BusinessCard \
     CalculatorFunctionalTests \
-    CalendarTests \
     camera_client_test \
     camera_metadata_tests \
     CellBroadcastReceiverTests \
